codegen/src/main/java/software/amazon/awssdk/codegen/model/config/customization/CustomizationConfig.java

```diff
@@ -572,4 +574,12 @@ public List<String> getInterceptors() {
     public void setInterceptors(List<String> interceptors) {
         this.interceptors = interceptors;
     }
+    
+    public Map<String, Boolean> getEnabledTraitValidations() {
+        return enabledTraitValidations;
+    }
+
+    public void setEnabledTraitValidations(Map<String, Boolean> enabledTraitValidations) {
+        this.enabledTraitValidations = enabledTraitValidations;
+    }
```

Comment:
Can we add docs or naming changes to make it clear that the required trait is only enforced for query strings? Or can we make it apply to everything marked as required?

I'm trying to avoid someone reading the code or customizations in the future thinking that enabling this flag will enforce the required trait for everything that is marked as required.

Reply:
Making change so that the RequiredTrait applies to all members marked with it.